ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -14,31 +14,37 @@ server {
     server_tokens off;
     log_not_found off;
 
+    # We gzip the responses that NGINX does for the following mime-types. We don't use gzip_static as otherwise it would
+    # request a pre-gzip version of each file on the file system, which would hammer the file system with requqests.
     gzip on;
     gzip_disable "MSIE [1-6]\.";
     gzip_types text/plain text/css application/javascript text/xml application/xml application/xml+rss image/svg+xml;
 
-    open_file_cache max=10000 inactive=300s;
+    # These directives are used to prevent our server to continuously attempt to open the same requested files on the file_system
+    # W use this so that file_system accesses get cached during this period of time.
+    # For specific Location directives, such as /dist we also cache the 404's and any other Server Error
+    open_file_cache max=100000 inactive=300s;
     open_file_cache_valid 120s;
     open_file_cache_min_uses 2;
     open_file_cache_errors off;
 
     root /home/www/nodejs;
     default_type text/plain;
     index index.html;
+
+    # We set a default cache of 1 hour and a max age of 4 hours
```

Comment:
```suggestion
    # We set a default cache for browsers of 1 hour, and ask Cloudflare to cache for 4 hours
```

Reply:
Should we set both to 14400? Might be less confusing since CF overrides max-age to 14400.